csi.proto

```diff
@@ -802,6 +802,14 @@ message ListVolumesRequest {
 message ListVolumesResponse {
   message Entry {
     Volume volume = 1;
+
+    // A list of all `node_id` of nodes that the volume in this entry
+    // is controller published on.
+    // This is a REQUIRED field if the SP has the
+    // LIST_VOLUMES_PUBLISHED_NODES controller capability. If the SP
+    // does not have the LIST_VOLUMES_PUBLISHED_NODES capability this
+    // field MUST be empty.
+    repeated string published_nodes = 2;
```

Comment:
Should this list be here or part of `Volume` type? I think it belongs to `Volume` type rather than here. My reasoning for that is - if another request type possibly needs this field it can use it too.

Reply:
Thanks for bringing this up! We had originally discussed this as well but determined `Volume` contains `Spec`-type information that is an unchanging description of the object, whereas `published_nodes` doesn't really fit in there as a dynamic value.

There are also no other messages that use `Volume` besides the `CreateVolume` responss - and in that case the `published_nodes` should always be nil since the volume was just created

Reply:
ack. It seems reasonable.